### PR TITLE
Fix links to Microsoft Learn + Minor Formatting Updates

### PR DESCRIPTION
**Pull request by ckittel**
* Fixed links to Microsoft Learn
  * `docs.microsoft.com` -> `learn.microsoft.com`
  * remove `en-us`
  * resolve redirects
* AKS workshop is no longer a thing.  Remove links to it.  @mosabami the repo behind it <https://github.com/MicrosoftDocs/mslearn-aks-workshop-ratings-api> is subject to delete as it is no longer maintained.  If you use it, please bring in a copy to your repo.  This dependency could break any moment.
* Fixed some formatting issues I saw along the way

No functional changes.

### CODE REVIEW

File: Scenarios/AKS-Secure-Baseline-PrivateCluster/Bicep/02-AAD/ad_groups.ps1 [hunk 1]
```diff
@@ -16,5 +16,5 @@ if($isInstalled){
     New-AzADGroup -DisplayName $aksops -MailNickname $aksops
 }
 else {
-    Write-Output "Azuer Powershell not installed. Installation steps in: https://docs.microsoft.com/en-us/powershell/azure/install-az-ps"
+    Write-Output "Azuer Powershell not installed. Installation steps in: https://learn.microsoft.com/powershell/azure/install-az-ps"
```
Comment (Gordonby) [inline Scenarios/AKS-Secure-Baseline-PrivateCluster/Bicep/02-AAD/ad_groups.ps1:19]: Azure is typo'd here. May as well fix :)

**Review by Gordonby — APPROVED** [review]
Some great uplifts in here Chad.

**Review by mosabami — APPROVED** [review]
great pr

File: Scenarios/AKS-Secure-Baseline-PrivateCluster/Bicep/02-AAD/ad_groups.ps1 (hunk 1 above)
Comment (ckittel) [reply Scenarios/AKS-Secure-Baseline-PrivateCluster/Bicep/02-AAD/ad_groups.ps1:19]:
```suggestion
    Write-Output "Azure PowerShell not installed. Installation steps in: https://learn.microsoft.com/powershell/azure/install-az-ps"
```

**Pull request merged**